docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
 For more information about labels, refer to [apply custom
 metadata](../../userguide/labels-custom-metadata.md).
 
+### Volumes and bind-mounts (--mount)
+
+Services use a more advanced notation to 
+
+| Option                | Required          | Description
+|:----------------------|:------------------|:--------------------------------------------------------------------------------------------------------------------
+| **type**              |                   | The type of mount, can be either "bind", to bind-mount a directory or file from the host, or "volume" to use a managed volume. Defaults to "volume" if no type is specified.
+| **src**               | depends on type   | In case of a bind-mount (`"type=bind"`), this option is required, and specifies the path on the host that is bind mounted (e.g., `src=/path/on/host/`). When using a bind-mount (`"type=bind"`), the `src` path must be specified as an absolute path, and *must* be a pre-existing path, or an error is produced.<br /><br />For volumes (`"type=volume"`), this option is optional, and specifies the name of the volume (e.g., `src=my-volume`). If a volume with the given name does not exist, it is automatically created. If this option is omitted for a volume (`type=volume`), an ephemeral volume with a random name is generated. Random names are guaranteed to be unique on the host, but may not be unique cluster-wide. Ephemeral volumes have the same lifecycle as the container it is attached to, and are destroyed when the *container* is destroyed (which is upon `service update`, or when scaling or re-balancing the service).
```

Comment:
Required: `bind` only.

then similar to `type`
`bind`:  specifies the path on the host to bind mount into task containers. Must be a pre-existing absolute path. For example: ``src=/path/on/host/`.

`volume`: specifies the name of the volume. If the volume doesn't exists, the swarm automatically creates it. If you specify a `volume` mount, but do not include a `src`, the swarm creates an ephemeral volume with a random name. Random names are only guaranteed to be unique in the scope of the host (node?), not cluster-wide. Ephemeral volumes have the same lifecycle as the container it is attached to, and are destroyed when the *container* is destroyed. (we probably need a document somewhere on container lifecycle in the swarm)